src/event_loop.rs

```diff
@@ -296,6 +300,19 @@ impl<T> EventLoopWindowTarget<T> {
     #[cfg(any(target_os = "linux", target_os = "macos"))]
     self.p.set_progress_bar(_progress)
   }
+
+  /// Sets the theme for this window.
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Linux / macOS**: Theme is app-wide and not specific to this window.
+  /// - **Windows:** Unsupported. Use the set_theme function available in Window (Windows can have different themes for different windows)
```

Comment:
This is not entirely true, we have a state for perferred theme here https://github.com/Legend-Master/tao/blob/c626141feeda2a0f7e703b6501a02a46c99d619f/src/platform_impl/windows/event_loop.rs#L109 and this function could change this state.

Reply:
Is there anyway we can get the windows from here? Just changing the state feels incomplete to be honest

Reply:
Same as you did with emitting event, you will need to use `PostMessageW`

Reply:
I mean we need the `hwnd` or else how can we make the existing windows change their theme?

Reply:
Posting `WM_SETTINGCHANGE` message to the `HWND` in `EventLoopWindowTarget` should trigger this line https://github.com/tauri-apps/tao/blob/c626141feeda2a0f7e703b6501a02a46c99d619f/src/platform_impl/windows/event_loop.rs#L2072

Reply:
I think the `preferred_theme` here is a clone?

Reply:
Maybe just document this will only affect new windows?

Reply:
but why? we have a way to update existing windows as well.

Reply:
> I think the `preferred_theme` here is a clone of `EventLoopWindowTarget::preferred_theme`?

we can change it to use `Mutex` and clone that mutex instead, so we can change the state later

Reply:
that ofc depends on if `PostMessageW(null, WM_SETTINGCHANGE)` actually triggers our `WM_SETTINGCHANGE` handler for each window

Reply:
Got it, made the change